src/components/auth/PublicRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+import {Route, Redirect, RouteProps} from 'react-router-dom';
+import {RootState} from '../../store';
+
+import AppPaths from '../../const/paths';
+
+interface Props extends RouteProps {
+  component: any;
+}
+
+export const PublicRoute = ({component: Component, ...rest}:Props) => {
+  const {authenticated} = useSelector((state: RootState) => state.auth);
```

Comment:
Same thing here unless you need the entirety of the auth state it's probably best to just point to the `state.auth.authenticated` boolean.

Reply:
Same answer to same question asked above!